BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -66,11 +66,7 @@ def create(args:,
              weather:)
     @schedules = {}
 
-    if @column_names.nil?
-      @column_names = SchedulesFile::Columns.values.map { |c| c.name }
-    end
-
-    invalid_columns = (@column_names - SchedulesFile::Columns.values.map { |c| c.name })
+    invalid_columns = (@column_names - ScheduleGenerator.export_columns)
```

Comment:
All of these references to `SchedulesFile::Columns` should really reference `export_columns` (which only looks at columns that can be stochastically generated). This is why we didn't notice that `electric_vehicles` was missing from the list of possible choices for the Column Names argument.